### DISCUSSION TITLE
Social sign up is not working well.

### DISCUSSION THREAD

**Discussion by Super0109** [community/community#60995] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, everyone. hope you are doing well.
Now, I am doing google sign up using with React-Native.
But that is not working well properly.
I am very thankful if you can help this problem.

Thanks in advance.

**Comment by SuperSmile0109** [comment, marked as answer]
Hello,
I have solved this problem using these steps:

1). Goto https://console.developers.google.com/ and delete (Android Client) if it is created.
2). Click on Create Credential and click on OAuthClientID and select android
3). copy and paste your SHA 1 fingerprint
4). type your package name then saved
5). Goto console.firebase.com
6). download google-service.json move it into your project into app directory
7). Open google-service.json and find out client id type 3 and use it as default_web_client_id
8). Run your App

I hope that is working on your side.
Best regards

**Reply by Super0109** [reply]
Hi, thanks for your support

I followed your steps and then maybe I missed one step.
Btw, now that is working now.

Thank you very much
